_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -52248,6 +52252,18 @@
 	icon_state = "platingdmg3"
 	},
 /area/maintenance/department/security/brig)
+"tSA" = (
+/obj/structure/cable,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/scrubbers/hidden/layer2,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4{
+	dir = 4
+	},
+/obj/machinery/door/poddoor/shutters/preopen{
+	id = "chemistry_shutters";
+	name = "chemistry shutters"
```

Comment:
Again, capital letters.

Reply:
Does it LOOK like i EVEN know how to do that alright HOW ABOUT YOU tell me HOW to do that and I'LL do it but ILL SAY i did it by myself

Reply:
Right click the tile where the shutters are in StrongDMM > Select the shutters > Edit > Search for the name var > Change it

Reply:
There it should be done...?